.gitignore

```diff
@@ -1,5 +1,18 @@
-/target/
-/.settings/
-/.classpath
-/.project
-/bot.properties
+# Maven related
+target/
+
+# Javadoc dir
+javadocs/
+
+# IntelliJ Idea related
+.idea/
+*.iml
+
+# Eclipse related
+.settings/
+.classpath
+.project
+
+# Miscellenous
+*.properties
+*.log
```

Comment:
I think it makes sense to leave it in, because the program will generate .log files if you run it from your development environment.